lib/vertex_client/responses/quotation_fallback.rb

```diff
@@ -11,7 +11,7 @@ def subtotal
       end
 
       def total_tax
-        @total_tax ||= line_items.sum(&:total_tax).floor(2)
+        @total_tax ||= line_items.sum(&:total_tax).round(2, :half_even)
```

Comment:
Do we return Floats or BigDecimal here?

Reply:
`BigDecimal` as the `line_items` are mapped into `LineItem` where we initialize the `total_tax` as a `BigDecimal`.  I think the `round(2, :half_even)` wouldn't even work on `Float` as the argument structure for `Float.round` is different (as far as I recall correctly)